src/Notifier.js

```diff
@@ -123,6 +123,26 @@ var Notifier = {
         return global.Notification.permission == 'granted';
     },
 
+    isPermissionDefault: function() {
+        if (!this.supportsDesktopNotifications()) return false;
+        return global.Notification.permission == 'default';
+    },
+
+    showToolbar: function() {
```

Comment:
This doesn't seem to be used anywhere. Is there another PR to come?

`showToolbar` sounds like it will show the toolbar. This should be `shouldShowToolbar` or similar (and it needs a comment saying what it does)